.github/workflows/dependabot-post-update.yml

```
@@ -0,0 +1,54 @@
+name: dependabot-post-update
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)